feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go

```diff
@@ -230,111 +230,153 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
 
+	t.Logf("Create qos forwarding groups config")
+	forwardingGroups := []struct {
+		desc         string
+		queueName    string
+		targetGroup  string
```

Comment:
The extraneous space between the field name and type throughout is causing the gofmt check to fail. Please fix.